corehq/apps/registry/models.py

```diff
@@ -0,0 +1,127 @@
+from datetime import datetime
+
+from autoslug import AutoSlugField
+from django.contrib.auth.models import User
+from django.contrib.postgres.fields import JSONField, ArrayField
+from django.db import models
+from django.utils.text import slugify
+
+from corehq.apps.domain.utils import domain_name_stop_words
+from corehq.apps.registry.exceptions import RegistryAccessDenied
+
+
+def slugify_remove_stops(text):
+    words = slugify(text).split('-')
+    stop_words = domain_name_stop_words()
+    return "-".join([word for word in words if word not in stop_words])
+
+
+class RegistryManager(models.Manager):
+
+    def owned_by_domain(self, domain, is_active=None):
```

Comment:
I think there are enough places where we want to include inactive registries that it doesn't make sense exclude them by default.

It may turn out that we should but I'll wait until it's clearer how we are going to handle inactive registries in the downstream features.